### DISCUSSION TITLE
🐛 [BUG] 双轴图slider缩略轴update效果错误

### DISCUSSION THREAD

**Discussion by xuhailin** [antvis/G2Plot#3560] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20624418/233529677-59c4d6fa-1af4-47ef-b80b-f98a4f0aeb4b.png)

版本 2.4.25
如图： 使用update方法更新options，展示效果不对

**Comment by controllerz** [comment]
期望的效果什么样， 代码片段也没有。这种提bug，谁能给你解？

**Comment by xuhailin** [comment, marked as answer]
啊，不好意思，是我疏忽啦

我在示例上试了一下，sync默认是true，我没有配置为false。这个问题是我配置的错误。
```
  meta: {
    time: {
      sync: false, // 开启之后 slider 无法重绘
    },
  },
```

还有一个问题是拖动窗口缩略轴不能自适应，麻烦再看看

https://g2plot.antv.antgroup.com/zh/examples/dual-axes/column-line/#slider-column-line

![企业微信截图_d54443ff-1409-4eff-931d-d843b6d84959](https://user-images.githubusercontent.com/20624418/233890923-f63134ef-f395-4d2b-9379-47fb8ed843c1.png)
